### PR TITLE
First draft of organization documentation

### PR DESCRIPTION
**Pull request by mlocher**
Codeship will soon support organizations, here are the docs :sparkles:

### CODE REVIEW

File: _posts/administration/2015-07-14-organizations.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - project
+  - organizations
+  - team management
+categories:
+  - administration
+---
+
+Organizations simplify and enhance team management as well as subscription management for (larger) teams on Codeship.
+
+You can define arbitrary teams, add Codeship accounts to those teams and simply add the team to any organization project. You can now also define read-only access for a specific team to your projects.
```
Comment (mcgaffin) [inline _posts/administration/2015-07-14-organizations.md]: As I read this the order of actions is:
1. define team,
1. add accounts to team
1. add the team to an organization
but you need to add the team to the organization first, I think.

So, how about:
> You can define arbitrary teams and add them to any organization project and add Codeship accounts to those teams.

Maybe the intent here is not to specify the order, so maybe it's ok as is. :)

File: _posts/administration/2015-07-14-organizations.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+* include a table of contents
+{:toc}
+
+We currently offer the following roles (though more roles are already on our todo list):
+
+* **Owners** have control over all aspects of an organization. From changing the subscription to managing organization projects and teams.
+* **Managers** have control over team and project management of organization. They can add and remove projects and manage the organization teams by adding new team members or assigning projects to teams. They have access to all project and are able to change the project configuration
```
Comment (mcgaffin) [inline _posts/administration/2015-07-14-organizations.md]: * "of organization" => "of an organization"
* "access to all project" => "access to all projects"
* "project configuration" => "project configuration."

File: _posts/administration/2015-07-14-organizations.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+Two teams are created for each organization by default:
+* _Owners_, containing only the user who created the organization by default. You can however add any other Codeship account to the _Owners_ team as well.
+* _Managers_, containing nobody by default.
+
+If you want to create a new team, click the _Create new team_ button and select the appropriate role.
+
+![Creating a Team]({{ site.baseurl }}/images/administration/create_team.png)
+
+Once you have created a new team, you can add new team members via their email address as well as any already existing projects.
```
Comment (mcgaffin) [inline _posts/administration/2015-07-14-organizations.md]: "any already existing projects." => "any existing projects."

File: _posts/administration/2015-07-14-organizations.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+If you need to change the team settings (e.g. the name or the role), hover over the team card and click the gear icon showing on the right hand side.
```
Comment (mcgaffin) [inline _posts/administration/2015-07-14-organizations.md]: "(e.g. the name or the role)" => "(e.g., the name or the role)"

File: _posts/administration/2015-07-14-organizations.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+## Adding projects
+
+You can either add a project via the _Select project_ dropdown at the top, or via the _Create a new project_ button in the organizations project settings. Please make sure the correct account (either organization or your private account) is selected first, as it's currently not possible to transfer projects to another account.
```
Comment (mcgaffin) [inline _posts/administration/2015-07-14-organizations.md]: "organizations project settings" => "organization's project settings"

File: _posts/administration/2015-07-14-organizations.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+Once the project is created you can add it to any of your teams . Members of the _Owners_ and _Managers_ team will have access to all projects by default.
```
Comment (mcgaffin) [inline _posts/administration/2015-07-14-organizations.md]: "Once the project is created" => "Once the project is created,"

**Comment by mcgaffin** [comment]
nice doc, marko. My suggestions are just minor syntax and punctuation changes.

File: _posts/administration/2015-07-14-organizations.md (hunk 4 above)
Comment (mlocher) [inline _posts/administration/2015-07-14-organizations.md]: I'll probably never remember this one :D

**Comment by mlocher** [comment]
Thx, updated with your suggestions!

**Pull request merged**